vtr_flow/tasks/regression_tests/vtr_reg_weekly/koios/config/config.txt

```diff
@@ -22,6 +25,9 @@ circuit_list_add=dla_like.medium.v
 # Add architectures to list to sweep
 arch_list_add=k6FracN10LB_mem20K_complexDSP_customSB_22nm.xml
 
+# Add include files to the list
```

Comment:
Good to comment that this will use the advanced dsp block mode

Reply:
Adding the comment